Ja Navey

JSNavey opened the pull request.

JSNavey committed:
completed GET with id, DELETE and PUT

enitchals commented on server.js:
```diff
@@ -0,0 +1,76 @@
+server.post('/api/users', (req, res) => {
+    const { name, bio } = req.body;
+    db
+        .insert({ name, bio })
```
Great start here! Consider implementing a check to make sure that both `name` and `bio` values exist. From the readme:

> If the request body is missing the name or bio property:
> cancel the request.
> respond with HTTP status code 400 (Bad Request).
> return the following JSON response: { errorMessage: "Please provide name and bio for the user." }.

JSNavey committed:
added error msg to GET and POST

JSNavey committed:
added error messages to GET, POST and DELETE however, still working on the PUT method

JSNavey committed:
created client app and installed axios

JSNavey committed:
added cors and implemented GET to fetch the data.  Created boiler plates for friendsList and form components

JSNavey committed:
built add functionality but recieved error500

JSNavey committed:
Add form is functionality.

moved handleInput to AddForm and fixed addUsers function by adding get and then inside post

designerexpert commented:
# Comments.
* Continue committing often and using meaningful commit messages.
* Good job trying your hand at the stretch. At least now you can begin to see the full circle of front end and back end come together.
* I think you did a great job implementing a user interface for the back end server.
* Always remember when in doubt, start over from scratch and see how far you get, in order to test your knowledge and understanding, and getting in more repetitions.

JSNavey committed:
added error msg to PUT

JSNavey committed:
added error mes reaponse to PUT in server.js

pls disregard the earlier commit, pushed to wrong directory
